util/classFilter.js

```diff
@@ -0,0 +1,8 @@
+// pass in the class number and the db connection (which includes the escapte function)
+module.exports.classFilter = (classNum, db) => {
+    let classFilter = '';
+    if (classNum) {
+        classFilter = ' AND class = ' + db.escape(classNum);
+    }
+    return classFilter;
```

Comment:
Since all you're doing is just returning the string, I would write this lambda as follows (enclosing braces omitted):

```javascript
if (classNum) {
        return ' AND class = ' + db.escape(classNum);
} else {
        return '';
}

Reply:
Yes, I definitely agree!